.github/workflows/publish.yml

```diff
@@ -1,8 +1,11 @@
 name: Publish Package to npmjs
 on:
-  release:
-    types: [published]
   workflow_dispatch:
+    inputs:
+      version:
+        description: "The version to publish"
+        required: true
```

Comment:
This workflow now needs to be manually triggered from the Github Actions UI, as we need to be very specific about the version that will be applied to all packages (and parsing some release title for a version number might turn out flimsy).

This also allows us to publish under a certain tag.

Reply:
Would it be worth spending some effort getting changesets integrated? It handles workspaces with dependent packages pretty well from what I've seen (check our brand repo for an example, specifically the tailwind package that depends on the base `brand` package)